test/other/embind_tsgen_main.ts

```diff
@@ -1,30 +1,36 @@
 // Example TS program that consumes the emscripten-generated module to to
 // illustrate how the type definitions are used and test they are workings as
 // expected.
-import moduleFactory from './embind_tsgen.mjs';
+import moduleFactory from './embind_tsgen.js';
```

Comment:
Should we not be testing both of these options?

Reply:
I see.. so this `.js` file can be either one, depending on the test?

Perhaps add a comment to that effect here?